### ISSUE TITLE
POST gauge/gauge-api/requests/metrics 419 (unknown status)

### ISSUE THREAD

**Issue by mokhosh** [tobiasdierich/gauge#1]
Thanks for the package.
Doesn't show the live charts, has this error in the console.
```
POST gauge/gauge-api/requests/metrics 419 (unknown status)
gauge:1 Uncaught (in promise) SyntaxError: Unexpected token < in JSON at position 0
```

**Comment by tobiasdierich** [comment]
Thank for letting me know! Turns out I forgot to implement CSRF token handling. I'll fix this tomorrow. For now, you can add the `/gauge/*` routes to the CSRF exceptions inside the `app/Http/Middleware/VerifyCsrfToken.php` file:

```php
protected $except = [
   'gauge/*'
];
```

**Comment by tobiasdierich** [comment]
I've implemented CSRF hanlding in 7cac0dc and tagged a new release.
